test/model/test_node.py

```diff
@@ -998,3 +1004,168 @@ async def test_get_highest_security_class(multisensor_6: Node, uuid4, mock_comma
         "nodeId": node.node_id,
         "messageId": uuid4,
     }
+
+
+async def test_test_powerlevel(multisensor_6: Node, uuid4, mock_command):
+    """Test node.test_powerlevel command."""
```

Comment:
```suggestion
    """Test node.test_power_level command."""
```

Reply:
this is incorrect, the command on the server side is still `node.test_powerlevel`